src/RestServer/Controllers/v1/WalletController.cs

```diff
@@ -0,0 +1,646 @@
+        /// <returns>An array of wallet address objects.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("{session:required}/address/list", Name = "GetWalletListAddress")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletAddressModel[]))]
+        public IActionResult WalletListAddresses(
+            [FromRoute(Name = "session")]
+            Guid sessionId)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var accounts = new List<WalletAddressModel>();
+            foreach (var account in wallet.GetAccounts())
+                accounts.Add(new WalletAddressModel()
+                {
+                    Address = account.Address,
+                    ScriptHash = account.ScriptHash,
+                    Publickey = account.GetKey().PublicKey,
+                    HasKey = account.HasKey,
+                    Label = account.Label,
+                    WatchOnly = account.WatchOnly,
+                });
+            return Ok(accounts);
+        }
+
+        /// <summary>
+        /// Deletes an account from the wallet.
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <param name="scriptHash" example="0xed7cc6f5f2dd842d384f254bc0c2d58fb69a4761">ScriptHash of the wallet address.</param>
+        /// <returns>Empty body response.</returns>
+        /// <remarks>No backups are made.</remarks>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("{session:required}/delete/{account:required}", Name = "WalletDeleteAccountByAddressOrScriptHash")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public IActionResult WalletDeleteAccount(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromRoute(Name = "account")]
+            UInt160 scriptHash)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            if (wallet.DeleteAccount(scriptHash) == false)
+                throw new WalletException($"Could not delete '{scriptHash}' account.");
+            else
+            {
+                if (wallet is NEP6Wallet wallet6)
+                    wallet6.Save();
+                return Ok();
```

Comment:
```suggestion
                return Ok("Address is deleted.");
```
Should return something I think, otherwise I don't know if my request is correct...

Reply:
Its Rest, so it just returns json models and http status. Just something we need to document.

Reply:
because if we do that then the response `content-type` won't be correct.

Reply:
I see